e2e/staticpvc.go

```diff
@@ -140,7 +142,11 @@ func validateRBDStaticPV(f *framework.Framework, appPath string, isBlock bool) e
 
 	app.Namespace = namespace
 	app.Spec.Volumes[0].PersistentVolumeClaim.ClaimName = pvcName
-	err = createApp(f.ClientSet, app, deployTimeout)
+	if checkImgFeat {
+		err = createAppErr(f.ClientSet, app, deployTimeout, "missing required parameter imageFeatures")
```

Comment:
keep the timeout less. so that CI can pass early? and make the error message we are checking as an argument.

Reply:
>keep the timeout less. so that CI can pass early? 

The function will exit as soon as the error msg is seen in the event i.e,. when nodeplugin respond for mount req, so it should'nt cause any delay for *passing* ci. 

>and make the error message we are checking as an argument.

I will send a follow up pr to cleanup and refactor functions better to address this and @nixpanic suggestion [here](https://github.com/ceph/ceph-csi/pull/2139#discussion_r655382522)